From 3fb0ec972acb933ec01a291e0930c8f15fe0f9e8 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Wed, 27 Feb 2019 10:15:41 -0800
Subject: [PATCH 0517/1852] dt-bindings: display: xlnx: Add clock framework
 support to VPSS CSC bridge driver

commit 1562e81b4922f033dcd73830fae2a53745c0eaff from
https://github.com/Xilinx/linux-xlnx.git

This patch adds support for clock framework to VPSS CSC bridge driver by
adding and making clock as required device tree property.

This breaks backwards compatibility as the IP version hasn't changed.
So compatible string isn't changed. But new required clock property is
added.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/display/xlnx/xlnx,vpss-csc.txt         | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-csc.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-csc.txt
index 7991e3e96937..29190b9d0fb3 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-csc.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-csc.txt
@@ -15,10 +15,13 @@ Required properties:
 
 -reset-gpios: GPIO specifier to assert/de-assert the reset line.
 
+- clocks: phandle to IP clock.
+
 Example:
 	csc@a0040000 {
 		compatible = "xlnx,vpss-csc";
 		reg = <0x0 0xa0040000 0x0 0x10000>;
 		reset-gpios = <&gpio 0x0 GPIO_ACTIVE_LOW>;
 		xlnx,video-width = <8>;
+		clocks = <&misc_clk_0>;
 	}
-- 
2.31.1

